### PR TITLE
Delete autogenerated test stubs

### PR DESCRIPTION
**Pull request by maxb**
I propose that we delete the autogenerated test stubs.

They are only stubs, containing commented out code to serve as
a starting point for human written tests - they do not actually test
anything.

I suggest that it is unlikely that the project will ever make use of
these test stubs for the purpose intended by the generator - I do not
think anyone is going to be going through the API manually filling in
tests that exercise all/most client methods against a Vault server.

Additionally, in the case of the `src/Vault.Test/Api/*Tests.cs` files,
they are generated by openapi-generator only if they do not already
exist, and as a result are already out of date with various method
renames that have occurred, as OpenAPI operation IDs have been tweaked
over time.

I have retained one file of non-generated tests.

This change harmonizes with the sibling vault-client-go project, which
already has generation of test stubs turned off.

### CODE REVIEW

**Comment by maxb** [comment]
This will also fix the "Unchanged files with check annotations" section showing up in every PR diff, complaining about unused code in the tests.

File: .openapi-generator-ignore [hunk 1]
```diff
@@ -1,34 +1,12 @@
...
 
-# You can recursively match patterns against a directory, file or extension with a double asterisk (**):
-#foo/**/qux
-# This matches foo/bar/qux, foo/baz/qux, and foo/bar/baz/qux
-
-# You can also negate patterns with an exclamation (!).
-# For example, you can ignore all files in a docs folder with the file extension .md:
-#docs/*.md
-# Then explicitly reverse the ignore rule for a single file:
-#!docs/README.md
```
Comment (maxb) [inline .openapi-generator-ignore:23]: I simplified this comment to match the one in sibling project vault-client-go.

**Review by averche — APPROVED** [review]
I agree. If we ever want to experiment with the generated tests, we can easily revert this :)

Thanks for the PR!

**Comment by maxb** [comment]
Conflict resolved.

**Review by AnPucel — APPROVED** [review]
Always happy to unnecessary code being removed!

**Pull request merged**